CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Next
 - Fix [#1862](https://github.com/Glavin001/atom-beautify/issues/1862) Add support for ocp-indent as an executable
 - See [#601](https://github.com/Glavin001/atom-beautify/issues/1862) Add support for tsx files
+- See [#2026](https://github.com/Glavin001/atom-beautify/issues/2026) Add Vue support to ESLint Fixer beautifier
```

Comment:
I think it's important to add a note here about requiring https://github.com/vuejs/eslint-plugin-vue for using ESLint Fixer on Vue files.